docs/Options.md

```diff
@@ -136,6 +137,8 @@ Directory is always shown and truncated to the value of `SPACESHIP_DIR_TRUNC`. W
 | `SPACESHIP_DIR_TRUNC` | `3` | Number of folders of cwd to show in prompt, 0 to show all |
 | `SPACESHIP_DIR_TRUNC_REPO` | `true` | While in `git` repo, show only root directory and folders inside it |
 | `SPACESHIP_DIR_COLOR` | `cyan` | Color of directory section |
+| `SPACESHIP_DIR_LOCK_SYMBOL` | `·🔒` | The symbol displayed if directory is write-protected |
```

Comment:
Please, use powerline lock icon.

Reply:
@denysdovhan just to confirm, have you seen [this thread](https://github.com/denysdovhan/spaceship-prompt/pull/340#discussion_r162799461)? I'm personally using powerline lock icon too, but I kinda agree with the argument there.

Reply:
We already require powerline patched font, why don't we use all of its power?

Reply:
Updated with powerline lock symbol